org.versotile.verso.yml

```diff
@@ -36,7 +36,9 @@ modules:
       # Build Verso release profile
       - cargo --offline fetch --manifest-path Cargo.toml
       - cargo --offline build --release --features flatpak
+      - cargo --offline build --release --package verso
       # Install Verso
+      - install -Dm755 ./target/release/versoview -t /app/bin/
       - install -Dm755 ./target/release/verso -t /app/bin/
```

Comment:
Will verso binary still be generated now? It's only versoview on my end, just want to make a double check if this one should exist.

Reply:
It will only be generated if you build the verso package, and instead of the old verso, it's now a demo launching the versoview with https://example.com and after 10 seconds, switching to https://docs.rs